elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -261,16 +270,14 @@ def get_assembly_bug_ids(runtime, bug_tracker_type):
 
 
 def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
-                            permitted_bug_ids, permissive=False, major_version: int = 4):
+                            permitted_bug_ids, operator_advisory, permissive=False, major_version: int = 4):
```

Comment:
I would make this `metadata_advisory_name` or something similar
since we usually consider "extras" as the operator advisory

Reply:
Does `operator_bundle_advisory` sound better?

Reply:
yeah that's good too

Reply:
added in https://github.com/openshift-eng/art-tools/pull/341/commits/65be8eaf67986030464a864aa735a1cd755bfdc9